### PR TITLE
Bump CI config for serving release-next to 4.10,4.11 from 4.8,4.9

### PR DESCRIPTION
**Pull request by nak3**
As per title, this patch bumps the CI config to 4.10,4.11 from 4.8,4.9.

### CODE REVIEW

File: ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-postsubmits.yaml [hunk 1]
```diff
@@ -10,10 +10,10 @@ postsubmits:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/is-promotion: "true"
-      ci-operator.openshift.io/variant: "412"
```
Comment (ReToCode) [inline ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-postsubmits.yaml:13]: Was this, and similar changes (12 -> 10), intended?

Comment (nak3) [reply ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-postsubmits.yaml:13]: I configured `ci-operator/config/*` but other files such as `ci-operator/jobs/*` are auto updated via `make jobs` (the script is owned by this repo's not ours).

So, this file is generated by `make jobs` automatically and it is not my intention but the automated script's intention.

Comment (ReToCode) [reply ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-postsubmits.yaml:13]: Ah yep, I see. Still confusing :)

**Pull request merged**